src/pollution_model/run_medusa_2.py

```diff
@@ -52,10 +52,50 @@ class SurfaceType(StrEnum):
     CAR_PARK = "CrP"  # CarParks are classified the same as roads
 
 
+ROOF_SURFACE_TYPES = {SurfaceType.CONCRETE_ROOF, SurfaceType.GALVANISED_ROOF, SurfaceType.COPPER_ROOF}
+ROAD_SURFACE_TYPES = {SurfaceType.ASPHALT_ROAD, SurfaceType.CAR_PARK}
+
+
+class MedusaRainfallEvent(NamedTuple):
+    """
+    Rainfall event parameters for MEDUSA 2.0 model.
+
+    Attributes
+    ----------
+    antecedent_dry_days: float
+        Length of antecedent dry period (days).
+    average_rain_intensity: float
+        Average rainfall intensity of the event (mm/h).
+    event_duration: float
+        Duration of the rainfall event (h).
+    rainfall_ph: float
+        The acidity level of the rainfall.
+    """
+
+    antecedent_dry_days: float
+    average_rain_intensity: float
+    event_duration: float
+    rainfall_ph: float
```

Comment:
Sam raised a point in discord that we're really expecting floats here rather than ints, and ints can be used in place of floats if the developer places one in without trouble but we're expecting floats.

There's lots of nuance to this discussion though, many different points are raised in this discussion I found: https://discuss.python.org/t/clarifying-the-float-int-complex-special-case/54018/12

I'm leaning towards only typehinting floats, since its more clear what our intention is.

Reply:
Yes, in the paper, they are all float values, so it makes sense to just use float here.

Integers can be stored as floats, but you lose precision when using floats for integer values.

My question is, if we also type hint this as an `int` and the user inputs an integer, I assume it would be stored as such. But if we only type hint this as a `float`, would it store the value as an integer or a float?

Additionally, for the front-end experience, since we are currently allowing the user to specify the rainfall parameter values, I thought it makes more sense to display `2 days` instead of `2.0 days` when the value is a whole number like 2. However, I’m not sure how this works on the front-end, so maybe it doesn't matter.